jpa/eclipselink.jpa.test.jse/src/it/java/org/eclipse/persistence/jpa/test/query/TestQuerySyntaxStringTests.java

```diff
@@ -211,4 +213,129 @@ public void testString_concatOperatorWhere02() {
             }
         }
     }
+
+    @Test
+    public void testString_replaceFunctionSelect01() {
+        if (emf == null)
+            return;
+
+        //REPLACE is not implemented in Apache Derby DB
+        if (emf.unwrap(Session.class).getPlatform().isDerby()) {
```

Comment:
can we get this emulated somehow? (at least file an issue so it can be investaged..)

Reply:
I tried to find any SQL function in Derby
https://db.apache.org/derby/docs/10.16/ref/
https://db.apache.org/derby/docs/10.16/ref/rrefsqlj29026.html
but I don't see there any String function which should search String within String or something near String replacement.

About emulation in `org.eclipse.persistence.platform.database.DerbyPlatform` like method `derbyPowerOperator()` I'm missing any basic String functions to do it as I mentioned.

Reply:
ok